modules/tensor_mechanics/src/materials/Compute2DIncrementalStrain.C

```diff
@@ -0,0 +1,52 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
```

Comment:
Stephanie is right. We don't have it in the standard so let's not enforce it for this PR. I know that we are inconsistent so we'll need to update the standard, then the code accordingly.